tests/integration/workflows/test_edit_article.py

```diff
@@ -66,6 +70,12 @@ def edit_workflow(workflow_app):
     eng_uuid = start('edit_article', data=factory.record_metadata.json)
     obj = WorkflowEngine.from_uuid(eng_uuid).objects[0]
 
+    # user id is required for edit article workflows
+    wf = workflow_object_class.get(obj.id)
+    wf.id_user = user.get_id()
```

Comment:
how it was working before?

Reply:
because user_id wasn't being used anywhere in the code. now it is.